### ISSUE TITLE
why is session from 0 to 9 in pre-trained stage?

### ISSUE THREAD

**Issue by Alex-arch** [icoz69/CEC-CVPR2021#17]
hello, nice work. I have some questions about the code. why is session from 0 to 9 in pre-train stage? All we need is a pre-trained model which could classify 60 base classes. What's more, in the session_*.txt, should we write the label such as 60,61,62.etc rather than some big number?

**Comment by Nan-S** [comment]
Yes, we only pre-train the 60 base classes in our pre-training stage. We use the original session_*.txt file from the [TOPIC ](https://github.com/xyutao/fscil). The big numbers are actually the img for the specific dataset. The dataloader file will help you translate each image to the corresponding img to class pair.
